holoviews/plotting/bokeh/__init__.py

```diff
@@ -191,11 +191,10 @@ def colormap_generator(palette):
 options.Ellipse = Options('style', color='black')
 options.Polygons = Options('style', color=Cycle(), line_color='black',
                            cmap=dflt_cmap)
-options.Rectangles = Options('style', cmap=dflt_cmap)
-options.Segments = Options('style', cmap=dflt_cmap)
 
 # Geometries
-options.Rectangles = Options('style', line_color='black')
+options.Rectangles = Options('style', color=Cycle(), cmap=dflt_cmap, line_color='black')
+options.Segments = Options('style', color='black', cmap=dflt_cmap)
```

Comment:
What's the motivation for having one of these be a cycle and the other be a fixed color?

Reply:
It looked like all the other "line-only" things like Segments are drawn in black, like error bars. I suppose it depends on the main use case.

For the Rectangles, I suppose it is an oversight when it was added, as the options were set twice with different values. Polygons sets a facecolor as well. OTH, things like Ellipse do not have any filling by default, so maybe Rectangles should also default to border-only in black.

Reply:
The current behavior is not aligned to anything else I tried though, since it is filled by default with blue. Elements like Ellipse are not filled by default at all.